src/data/Repository.js

```diff
@@ -1,34 +1,38 @@
-import routes from './API';
+import routes from "./API";
 
 /**
  * Responsible for communication with the backend
  */
- class Repository {
+class Repository {
+  uploadFiles(files) {
+    return files.map((file) => {
+      return this._encodeImage(file)
+        .then((encodedImage) =>
+          fetch("https://8uxam9kkod.execute-api.ap-southeast-2.amazonaws.com/Prod/owleye/", {
+            method: "POST",
+            body: encodedImage.split(",")[1],
+          })
+        )
+        .then((response) => response.json())
+        .then((json) => this._decodeImage(json["res_img"]))
+        .catch((error) => console.error(error));
+    });
+  }
 
-    uploadFiles(files) {
-        return files.map(file => {
-            return this._encodeImage(file)
-            .then(encodedImage => fetch('https://8uxam9kkod.execute-api.ap-southeast-2.amazonaws.com/Prod/owleye/', { method: "POST", body: encodedImage.split(',')[1] }))
-            .then(response => response.json())
-            .then(json => this._decodeImage(json["res_img"]))
-            .catch(error => console.error(error))
-        })
-    }
+  _encodeImage(image) {
+    return new Promise((resolve, reject) => {
+      const reader = new FileReader();
+      reader.onloadend = () => resolve(reader.result);
+      reader.onerror = reject;
+      reader.readAsDataURL(image);
+    });
+  }
 
-    _encodeImage(image) {
-        return new Promise((resolve, reject) => {
-            const reader = new FileReader();
-            reader.onloadend = () => resolve(reader.result);
-            reader.onerror = reject;
-            reader.readAsDataURL(image);
-        })
-    }
-
-    _decodeImage(imageAsString) {
-        const image = new Image();
-        image.src = `data:image/png;base64,${imageAsString}`;
-        return image;
-    }
+  _decodeImage(imageAsString) {
```

Comment:
I think you should add more documentation here regarding why `data:image/png;base64,` needed to be appended

Reply:
Done